src/Components/ScrivitoExtensions/SocialCardsTab.tsx

```diff
@@ -1,3 +1,5 @@
+// TODO: remove workaround for #11474
+/* eslint-disable @typescript-eslint/restrict-template-expressions */
```

Comment:
Why?

Reply:
From the docs:
> The severity is a string denoting the severity of the validation result and can have one of the following values: "error", "warning" and "info".

The current SDK will return `undefined` under some circumstances (and is typed accordingly), but this will be fixed upstream.